modules/@apostrophecms/notification/index.js

```diff
@@ -255,8 +259,8 @@ module.exports = {
               throw self.apos.error('invalid');
             }
             start = Date.now();
-            modifiedOnOrSince = req.query.modifiedOnOrSince && self.apos.launder.date(req.query.modifiedOnOrSince);
-            seenIds = req.query._id && self.apos.launder.ids(req.query._id) && req.query._id.split(',_id=');
+            modifiedOnOrSince = req.query.modifiedOnOrSince && new Date(req.query.modifiedOnOrSince);
```

Comment:
This should have its own try/catch and throw `self.apos.error('invalid')` so that it comes through as a 400 rather than a 500.

Reply:
(If the date string is bad)

Reply:
Done